smart-contracts/test/test_security.js

```diff
@@ -388,6 +419,11 @@ describe("Security Test", function () {
     });
 
     it("should allow users to unpeg troll token, but then does not receive", async function () {
+      // Add the token into white list
+      await state.bridgeBank.connect(operator)
+      .updateEthWhiteList(state.troll.address, true)
```

Comment:
Please change this whole file to indent statements that extend from previous lines.